pkg/auth/audit/audit.go

```diff
@@ -110,6 +135,13 @@ func (a *auditLog) write(userInfo *User, reqHeaders, resHeaders http.Header, res
 	a.log.RequestHeader = filterOutHeaders(reqHeaders, sensitiveRequestHeader)
 	a.log.ResponseHeader = filterOutHeaders(resHeaders, sensitiveResponseHeader)
 	a.log.ResponseCode = resCode
+	if a.log.UserLoginName != "" {
+		if a.log.User.Extra == nil {
+			a.log.User.Extra = make(map[string][]string)
+		}
+		a.log.User.Extra["username"] = []string{a.log.UserLoginName}
+		logrus.Debugf("Added username for login request to audit log %v", a.log.User)
```

Comment:
Do we want to print out the whole User object or just the added username + UserLoginName?

Reply:
changed the debug log to log the UserLoginName